tests/TestHarness/Node.py

```diff
@@ -550,6 +550,38 @@ def findInLog(self, searchStr):
                         return True
         return False
 
+    # verify only one or two 'Starting block' per block number unless block is restarted
+    def verifyStartingBlockMessages(self):
+        dataDir=Utils.getNodeDataDir(self.nodeId)
+        files=Node.findStderrFiles(dataDir)
+        for f in files:
+            blockNumbers = set()
+            duplicateBlockNumbers = set()
+            duplicatesFound = False
+            lastRestartBlockNum = 0
+
+            with open(f, 'r') as file:
+                for line in file:
+                    match = re.match(r".*Restarting exhausted speculative block #(\d+)", line)
+                    if match:
+                        lastRestartBlockNum = match.group(1)
```

Comment:
Can "continue" be put here inside the match? The rest of matches will not be true.